### ISSUE TITLE
Where are the tests?!

### ISSUE THREAD

**Issue by ariejan** [AikedeJongste/omnikassa2#1]
![trollface](https://user-images.githubusercontent.com/1913/49153407-f2501f00-f315-11e8-8a08-72f724f94621.png)

**Comment by ariejan** [comment]
Funny thing, I'm working on implementing OmniKassa 2-punt-0 as well :-) Care to collaborate on this?

**Comment by aikedejongste** [comment]
> Care to collaborate on this?

Yes please! I made this with @arnoutdemooij for a client. And as you can see this is very much a work in progress. I could use some help with the testing part. My problem is that there is no option in the Rabobank dashboard to specify a development/testing url to receive the notify posts on.

**Comment by bjornconstructors** [comment]
@ariejan http://www.expleo.nl/media/comingout.mp3
